editors/nano/Portfile

```diff
@@ -3,8 +3,7 @@
 PortSystem          1.0
 
 name                nano
-version             6.4
-revision            0
```

Comment:
Restored `revision`.

Reply:
Please let me know if you'd like me to squash both commits.

Reply:
Yes, that would be good.

Reply:
Done

Reply:
@reneeotten

Reply:
ehm, why am I mentioned here? Even though the port is `openmaintainer` we do allow the usual 72h for the maintainer to take a look before merging (especially since this version jump would typically be considered a major update). So if you expected me to merge this PR that's not going to happen - you can try and ping again after the usual three days ;)